<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>801</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="textTitle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>721</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:28px;
color:white;</string>
     </property>
     <property name="text">
      <string>Valitse talletettava summa ja paina Valmis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountInput">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;
border-radius:10px;
font-size:25px;
padding:10px 10px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConfirm">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #4CAF50;
background-image: url(../img/icon_confirm.png);
background-repeat: no-repeat;
background-position: right middle;
color: rgb(255, 255, 255);
border-radius: 15px;
padding:5 5px;
font-size:20px;
}
QPushButton:hover {
background-color: #43A047;
color: rgb(245, 245, 245);
}
</string>
     </property>
     <property name="text">
      <string>Valmis</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(130, 130, 130);
background-image: url(../img/icon_back.png);
background-repeat:no-repeat;
background-position:left middle;
color: rgb(255, 255, 255);
border-radius:15px;
padding:5 5px;
font-size:18px;
}
QPushButton:hover {
background-color: rgb(120, 120, 120);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Takaisin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:35px;
color:white;</string>
     </property>
     <property name="text">
      <string>â‚¬</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <widget class="QLabel" name="textTitle2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:28px;
color:white;</string>
     </property>
     <property name="text">
      <string>Talletus onnistui.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBack2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(130, 130, 130);
background-image: url(../img/icon_back.png);
background-repeat:no-repeat;
background-position:left middle;
color: rgb(255, 255, 255);
border-radius:15px;
padding:5 5px;
font-size:18px;
}
QPushButton:hover {
background-color: rgb(120, 120, 120);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Takaisin</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
